go.mod

```diff
@@ -18,6 +18,7 @@ require (
 	k8s.io/component-base v0.27.3
 	k8s.io/klog/v2 v2.90.1
 	k8s.io/kubernetes v1.27.3
+	k8s.io/pod-security-admission v0.0.0
```

Comment:
Is this the right tag? I see that the tags match OSS https://github.com/kubernetes/pod-security-admission/tags so should this be v0.27.3?

Reply:
The dependencies with "k8s.io/" prefix are replaced with v0.27.3 in go.mod, you can find in the last few lines in go.mod file.

Reply:
I think it would be nice to maintain some consistence because there are other dependencies that are v0.27.3